### ISSUE TITLE
Open Cypress not working with Cypress 10.3.0

### ISSUE THREAD

**Issue by Smriti-Suhag** [90poe/vscode-cy-helper#66]
**Describe the bug**
CypressHelper.commandForOpen is opening cypress as below:
npx cypress open --config testFiles=%path%
However with cypress update v10.3.0  **- testFiles is invalid now.**
Instead e2e.specPattern should be used.

**To Reproduce**
1. Click on open cypress lens above spec
2. Cypress opens but with error.

**Expected behavior**
To open the cypress without error.

**Environment (please complete the following information):**
- Extension version:                               v1.2.2
- VS Code version:                                1.67.1
- OS:                                                     Mac:

** Developer Tools Console Output **

**Additional context**

**Comment by Shelex** [comment]
Hi @Smriti-Suhag 
Thank you for opening an issue.
Due to publishing token expiration and missing option to regenerate new one I have forked plugin under my personal account:

[Cypress Helper v2](https://marketplace.visualstudio.com/items?itemName=shevtsov.vscode-cy-helper)
[Repository](https://github.com/Shelex/vscode-cy-helper)
In case you have found this extension via pack, it is also republished - [extension pack](https://marketplace.visualstudio.com/items?itemName=shevtsov.qa-extension-pack)

Support for cypress v10 is added in forked version of extension in v2.6.0

**Comment by Smriti-Suhag** [comment]
<img width="1440" alt="Screenshot 2022-07-27 at 1 38 23 PM" src="https://user-images.githubusercontent.com/108823112/181196855-464d7405-a460-42c3-8476-0959f739ea2b.png">
@Shelex the command is still using --testFiles. 😕

**Comment by Shelex** [comment]
@Smriti-Suhag 
Yes, to keep extension working for previous Cypress versions default command was not changed (`npx cypress open`). But if `cypressHelper.commandForOpen` configuration contains command with `--e2e` flag it treats it as cypress v10.

However, I agree that is not intuitive, so I have added an automatic `package.json` check in latest version (2.7.0) to detect spec flag name based on cypress version in dependencies or devDependencies.
